src/components/lyne-card-badge/lyne-card-badge.custom.d.ts

```diff
@@ -0,0 +1,4 @@
+export interface InterfaceCardBadgeAttributes {
+  size: 'regular' | 'small';
+  appearance: 'primary' | 'primary-negative';
```

Comment:
Mini mini detail: I could not find 'primary' mentioned in Figma. I suppose that you have used this to be able to differentiate the variants? In lyne-button we have used the property 'variant' for this. Not that it would make any difference, but since you seem to have followed the naming of the values we could also use the same property name?

Reply:
I could definitely use 'variant', but I struggle a bit with 'variant' since a variant holds for me also other data/content, here on the appearance/style changes. Same with the buttons. Maybe we rename the buttons from variant to appearance?

Reply:
As with @mbleuer and @feerglas discussed today we will use `appearance` only on new components and may introduce a renaming of properties (like for the button) during refactoring in the future which might cause breaking changes (but will be mentioned in the realese notes). 

For UX I proposed to adapt our naming in Figma.